scriptler/jobExecutedInDateRange.groovy

```diff
@@ -0,0 +1,30 @@
+/*** BEGIN META {
+ "name" : "List executed Jobs",
+ "comment" : "List Jobs executed in a range of time",
+ "core": "1.609",
+ "authors" : [
+ { name : "Kuisathaverat" }
+ ]
+ } END META**/
+
+ import jenkins.model.*
+ import java.util.Date
```

Comment:
Unused import. Especially if you remove the unused date below `Date now = new Date()`